### PR TITLE
fix conversion accuracy

### PR DESCRIPTION
**Pull request by zhangxiaoli73**
## What changes were proposed in this pull request?
1.  when running blas model with mkldnn backend, we should convert it to graph model first.
2.  running mean /variance of mkldnn batch_norm is not the same meaning in  mklblas, we  should divide running mean /variance with a scaler.

## How was this patch tested?

unit tests and example test

## Related links or issues (optional)
fixed https://github.com/intel-analytics/BigDL/issues/XXX

### CODE REVIEW

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/DnnGraph.scala [hunk 1]
```diff
@@ -481,6 +481,11 @@ class DnnGraph(
     )
     checkDuplicate()
   }
+
+  override def release(): Unit = {
+    super.release()
+    reorderManager.release()
```
Comment (i8run) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/DnnGraph.scala]: the `super.release()` will not release the whole memory.

Comment (zhangxiaoli73) [reply spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/DnnGraph.scala]: dnn graph is a container, and "super.release()" will release all modules.

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/models/resnet/TestImageNet.scala [hunk 2]
```diff
@@ -47,7 +47,7 @@ object TestImageNet {
       val sc = new SparkContext(conf)
       Engine.init
 
-      val model = Module.loadModule[Float](param.model)
+      val model = Module.load[Float](param.model)
```
[the remark this answers is not in the document]
Comment (zhangxiaoli73) [reply spark/dl/src/main/scala/com/intel/analytics/bigdl/models/resnet/TestImageNet.scala]: ok, changed

**Comment by zhangxiaoli73** [comment]
jenkins pass:
http://mlt-bddl01.sh.intel.com:8080/job/BigDL-PRVN-lite/456/

**Pull request merged**